geometry_msgs/Point position
geometry_msgs/Point goal
obstacleArray obstacles
---
bool success
geometry_msgs/Point projection